java_console/autotest/src/main/java/com/rusefi/simulator/SimulatorFunctionalTest.java

```diff
@@ -48,7 +49,11 @@ public void mainTestBody() throws InterruptedException {
         testOutputPin(bench_mode_e.HD_ACR2, Fields.BENCH_AC_RELAY_DURATION);
         testOutputPin(bench_mode_e.BENCH_AC_COMPRESSOR_RELAY, Fields.BENCH_AC_RELAY_DURATION);
         testOutputPin(bench_mode_e.BENCH_STARTER_ENABLE_RELAY, Fields.BENCH_STARTER_DURATION);
-// todo: fix me as well!        testOutputPin(bench_mode_e.BENCH_VVT0_VALVE, Fields.BENCH_VVT_DURATION);
+        testOutputPin(bench_mode_e.BENCH_VVT0_VALVE, Fields.BENCH_VVT_DURATION);
+
+        // check if VVT stops when rpm == 0
+        IoUtil.changeRpm(linkManager.getCommandQueue(), 0);
```

Comment:
looks like existing coverage using a different way to stop RPM

https://github.com/rusefi/rusefi/commit/1f81b10e82ca18744a046a2752d1dffbc9ab64a1#diff-0fd88a316753096797397c182963b3ee4c8acdf39a228d93425738ac948a2d9f